docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
In reply to Jerome's original comment/question about args possibly being hyphenated now, I switched to the develop branch and tried

```
$ python client_dev.py variantsets-search --datasetIds dataset1 http://localhost:8000/current
```

which worked, unsurprisingly, and

```
$ python client_dev.py variantsets-search --dataset-ids dataset1 http://localhost:8000/current
```

which failed.  So, if I understood your note correctly and am not over-generalizing from a single example, hyphenated args (`dataset-ids` for `datasetIds`) do not work.